src/pages/Signin.vue

```diff
@@ -0,0 +1,259 @@
+      />
+    </div>
+  </div>
+
+  <div
+    class="flex w-11/12 h-10 justify-evenly md:w-5/6 md:h-20 xl:w-3/4 mx-auto mt-20"
+  >
+    <template v-for="(image, index) in getGroupImages" :key="index">
+      <img :src="image" />
+    </template>
+  </div>
+
+  <div class="flex flex-col my-auto h-full pt-12 pb-10 space-y-3">
+    <!-- different input components -->
+    <div v-for="(authType, index) in getAuthTypes" :key="`id-${index}`">
+      <NumberEntry
+        v-if="isEntryNumber(authType)"
+        ref="numberEntry"
+        :label="numberEntryParameters.label"
+        :placeholder="numberEntryParameters.placeholder"
+        :isRequired="numberEntryParameters.required"
+        :maxLengthOfEntry="numberEntryParameters.maxLengthOfEntry"
+        :dbKey="numberEntryParameters.key"
+        @updateNumber="updateUserInformation"
+      />
+      <PhoneNumberEntry
+        v-if="isEntryPhoneNumber(authType)"
+        ref="phoneNumberEntry"
+        :label="phoneNumberEntryParameters.label"
+        :placeholder="phoneNumberEntryParameters.placeholder"
+        :isRequired="phoneNumberEntryParameters.required"
+        :dbKey="phoneNumberEntryParameters.key"
+        @updatePhoneNumber="updateUserInformation"
+      />
+      <Datepicker
+        v-if="isEntryDate(authType)"
+        ref="dateEntry"
+        :label="dateEntryParameters.label"
+        :isRequired="dateEntryParameters.required"
+        :dbKey="dateEntryParameters.key"
+        @updateDate="updateUserInformation"
+      />
+    </div>
+
+    <!-- invalid login message -->
+    <span
+      v-html="invalidLoginMessage"
+      v-if="isInvalidLoginMessageShown"
+      class="mx-auto text-red-700 text-base mb-1 text-center md:text-sm"
+    ></span>
+
+    <!-- submit button -->
+    <button
+      class="bg-primary hover:bg-primary-hover text-white font-bold shadow-xl uppercase text-lg mx-auto p-4 rounded disabled:opacity-50 btn"
+      data-cy="submitButton"
+      :disabled="isSubmitButtonDisabled"
+      @click="authenticate"
+    >
+      SIGN IN
+    </button>
+
+    <!-- signup button -->
+    <button
+      v-show="$store.state.sessionData.activateSignup"
+      @click="redirectToSignUp"
+      class="mx-auto pt-2 text-sm underline text-red-800"
+    >
+      If you are a new student, click here to register
+    </button>
+  </div>
+</template>
+<script>
+import useAssets from "@/assets/assets.js";
+import NumberEntry from "@/components/NumberEntry.vue";
+import Datepicker from "@/components/Datepicker.vue";
+import PhoneNumberEntry from "@/components/PhoneNumberEntry.vue";
+import { authToInputParameters } from "../services/authToInputParameters";
+import { validateUser } from "../services/validation.js";
+import { redirectToDestination } from "../services/redirectToDestination";
+import { sendSQSMessage } from "@/services/API/sqs";
+
+const assets = useAssets();
+export default {
+  name: "SignIn",
+  components: {
+    NumberEntry,
+    Datepicker,
+    PhoneNumberEntry,
```

Comment:
no textentry?

Reply:
`SignIn` will not have `TextEntry`, at least for now. Our only validators are those three. Until further is decided.